app/representers/api/image_representer.rb

```diff
@@ -17,19 +17,26 @@ def self_url(represented)
   end
 
   property :id, writeable: false
-  property :filename, writeable: false
+  property :filename
```

Comment:
I thought carrierwave would prevent direct update of this attribute, why this change?